### PR TITLE
tech: configure static files

### PR DESCRIPTION
**Pull request by rtio**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+deploy:
+	# composer install --prefer-dist --optimize-autoloader --no-dev
+	composer install
```
Comment (Luitame) [inline Makefile]: Is it for dev or prod?

File: config/packages/framework.yaml [hunk 2]
```diff
@@ -18,3 +18,7 @@ framework:
     serializer:
         enabled: true
         enable_annotations: true
+
+    assets:
+        base_urls:
+            - 'https://kdti-statics.s3.amazonaws.com/'
```
Comment (Luitame) [inline config/packages/framework.yaml]: What do you think about replacing this URL with an environment variable?

File: serverless.yml [hunk 3]
```diff
@@ -29,9 +32,54 @@ functions:
...
+        AssetsBucketPolicy:
+            Type: AWS::S3::BucketPolicy
+            Properties:
+                Bucket: !Ref Assets # References the bucket we defined above
+                PolicyDocument:
+                    Statement:
+                        -   Effect: Allow
+                            Principal: '*' # everyone
+                            Action: 's3:GetObject' # to read
+                            Resource: 'arn:aws:s3:::kdti-statics/*' # things in t
```
Comment (ianrodrigues) [inline serverless.yml]:
```suggestion
                            Resource: 'arn:aws:s3:::kdti-statics/*' # things in it
```

File: Makefile (hunk 1 above)
Comment (ianrodrigues) [reply Makefile]: I think we can have two commands:

`make deploy` or `make deploy:prod` for production
`make deploy:dev` for development

File: config/packages/framework.yaml (hunk 2 above)
Comment (ianrodrigues) [reply config/packages/framework.yaml]: I don't think it's necessary. We can only set in the other environment folders like:
* `packages/local/framework.yml`
* `packages/dev/framework.yml`

Comment (ianrodrigues) [reply config/packages/framework.yaml]: Actually, `local` should point to local directory. I suggest you to create an entry for that.

File: Makefile (hunk 1 above)
Comment (Luitame) [reply Makefile]: Agreed!

File: config/packages/framework.yaml (hunk 2 above)
Comment (Luitame) [reply config/packages/framework.yaml]: I got it. Great!

File: Makefile (hunk 1 above)
Comment (rtio) [reply Makefile]: my next PR does this

**Pull request merged**